code/modules/mob/living/say.dm

```diff
@@ -246,16 +246,35 @@ proc/get_radio_key_from_channel(var/channel)
 		var/randomnote = pick("\u2669", "\u266A", "\u266B")
 		message = "[randomnote] <span class='singing'>[message]</span> [randomnote]"
 
+	var/list/obj/item/used_radios = new //we'll come back to this later.
+
+
 	//handle nonverbal and sign languages here
 	if (speaking)
 		if (speaking.flags & NONVERBAL)
 			if (prob(30))
 				src.custom_emote(VISIBLE_MESSAGE, "[pick(speaking.signlang_verb)].")
 
 		if (speaking.flags & SIGNLANG)
+			if(istype(src,/mob/living/carbon/human)) //Allows for sign-to-speech over radio if radio-gloves with an active translator are worn.
```

Comment:
do not use `istype(src, ...)`

Reply:
I don't know what you want me to do instead.

Reply:
Find /mob/living/carbon/human/say(...) and add the sign language check on there instead.

Reply:
I just tried to do that and my way doesn't work.